### PR TITLE
improve UX of "Login as" page

### LINKED ISSUE #1466: Glitchy password masking when using "Login as..."
EE Version 6.1.4
PHP Version 7.3.30

Steps to recreate: 
1. Login as a Super Admin
2. Edit a member's roles and provide password to Authenticate & Save
3. Visit Administration > "Login as..." page of any member profile
4. Type in password to Login as... and see that the password masking is broken

_Note: 
I'm guessing there's a cookie that is caching the Super Admin's password, since after editing once member's roles, authenticating & saving, I'm not required to type my password in again to perform the action again or on another member. I find this inconsistent presence of the password field a little more alarming than helpful, and wonder if it's causing the glitchy password masking issue._

Screen Shot 2021-10-20 at 10 33 27 AM

### PR DESCRIPTION
**Pull request by intoeetive**
where URL input could be confused with password input

fixes #1466

EECORE-1464

fix the faded URL to Role Members on Roles page
related #1467

### CODE REVIEW

File: system/ee/language/english/myaccount_lang.php [hunk 1]
```diff
@@ -523,10 +523,6 @@
 
     'upload_avatar' => 'Upload — .gif, .jpg, .png (max %skb)',
 
-    'url' => 'Personal website',
-
-    'url_desc' => 'URL location of your personal website',
-
```
Comment (obfuscode) [inline system/ee/language/english/myaccount_lang.php:529]: Is this a conflict with the `Profile/Login.php` `lang('url')` field? Or is this related to this PR?

Comment (intoeetive) [reply system/ee/language/english/myaccount_lang.php:529]: Keeping these in place made the wrong string being output in profile/login (so yes, conflict)
I checked and could not find where it is used (including legacy profile pages and forum), I believe it has been removed from EE6

**Review by obfuscode — APPROVED** [review]
